op-deployer/pkg/deployer/opcm/implementations.go

```diff
@@ -22,7 +22,6 @@ type DeployImplementationsInput struct {
 	ProtocolVersionsProxy common.Address
 	SuperchainProxyAdmin  common.Address
 	UpgradeController     common.Address
-	UseInterop            bool // if true, deploy Interop implementations
```

Comment:
I believe this is safe to remove already; the L1 contract deployment code does not use this flag. And this is not the input to the L2Genesis script (which still does use a useInterop bool input, but should not use it, but can be addressed in follow-up issue, see other comment).